code/__defines/chemistry.dm

```diff
@@ -35,20 +35,19 @@
 
 // Some on_mob_life() procs check for alien races.
 #define IS_DIONA   1
-#define IS_VOX     2
-#define IS_SKRELL  4
-#define IS_UNATHI  8
-#define IS_TAJARA  16
-#define IS_XENOS   32
-#define IS_MACHINE 64
-#define IS_VAURCA  128
-#define IS_UNDEAD  256
+#define IS_SKRELL  2
```

Comment:
I'm not sure it's a good idea to re-number defines like this. Unless you are 100% sure that the magic value isn't used anywhere (as in, the defines are used in every relevant place) this could break something unexpectadly.